Cue/app/tabs/search/SearchListViewItem.js

```diff
@@ -0,0 +1,59 @@
+// @flow
+
+'use strict';
+
+import React from 'react'
+import { Image, View, Text, TouchableOpacity, Navigator } from 'react-native'
+
+import type { DeckMetaData } from '../../api/types'
+
+import CueColors from '../../common/CueColors'
+import CueIcons from '../../common/CueIcons'
+
+import DeckCardView from '../../common/DeckCardView'
+import DeckRating from '../../common/DeckRating'
+
+const styles = {
+  itemContainer: {
+    borderBottomWidth: 2,
+    borderColor: '#eee',
+    marginTop: 30,
+    flexDirection: 'row',
+    flex: 1,
+  },
+  textContainer: {
+    flex: 1,
+    flexDirection: 'column',
+    marginBottom: 30
+  },
+  firstText: {
+    fontSize: 16,
+    flex: 1,
+  },
+  cardContainer: {
+    marginLeft: 16,
+    marginRight: 16,
+    marginBottom: 30,
+  }
+}
+
+export default class SearchListViewItem extends React.Component {
+  props: {
+    navigator: Navigator,
+    deck: DeckMetaData
+  }
+
+  render() {
+    return (
+      <View style={styles.itemContainer}>

```

Comment:
I think we talked about this briefly today, but it would be great to be able to tap anywhere in the row to trigger the `onPress` (using `TouchableHighlight` on iOS and `TouchableNativeFeedback` on Android).